packages/scripts/src/cli.ts

```diff
@@ -18,7 +22,12 @@ void yargs(hideBin(process.argv))
                 demandOption: true
             }),
         async (argv) => {
-            await writeFernJsonSchema(resolve(cwd(), argv.filepath));
+            const filepath = argv.filepath;
+            const jsonSchema = zodToJsonSchema(DefinitionFileSchema, "Fern Definition");
+            const jsonSchemaStr = JSON.stringify(jsonSchema);
+            const config = (await prettier.resolveConfig(filepath)) ?? undefined;
+            const jsonSchemaFormatted = prettier.format(jsonSchemaStr, { ...config, filepath });
+            await writeFile(filepath, jsonSchemaFormatted);
```

Comment:
nit: I'd consider consolidating this into fewer lines to keep things a little more terse, but I get it if you like the readability as-is.

Reply:
i just kept the function the same as what it used to be (just copy pastad it)

Reply:
will just leave as is to keep things moving